### ISSUE TITLE
Sorting needed in the trade history blotter

### ISSUE THREAD

**Issue by ravichik** [Helix-ex/alpha#1]
Hi, After completing few trades, I have observed that the records don't show in a chronological order. I would expect to have sorting functionality enabled for all the columns in the blotter.

FYI, the funding blotter was showing in the chronological order.

See the screenshot

![image](https://github.com/Helix-ex/alpha/assets/135039321/6c209c92-35a0-4733-9629-b2409bde322d)

**Comment by morarasu-alexandru** [comment]
This should already be solved. @ravichik , can you please check and confirm?

**Comment by ravichik** [comment]
Yes, it is now sorted descending order. But do we create a sorting functionality for each column? Asecnding and Descending?
